<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced</class>
 <widget class="QDialog" name="Advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Дополнительные опции</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество происшествий с</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_date_s">
       <property name="placeholderText">
        <string> с</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> по </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_date_e">
       <property name="placeholderText">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_inc_count">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Лицо </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_reg_id">
       <property name="placeholderText">
        <string>регистрационный номер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>зарегистрирован в </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_pers_count">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>происшествиях</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="LE_inc_prot">
       <property name="placeholderText">
        <string>номер сообщения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_get_protocol">
       <property name="text">
        <string>Получить протокол по происшествию</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
